### ISSUE TITLE
UI Interaction Clarity

### ISSUE THREAD

**Issue by TheEclipsee** [alex4401/mediawiki-extensions-ThemeToggle#20]
Could be made more clear that the text is a dropdown and what its purpose is by inserting text such as Theme: before the theme name

**Comment by alex4401** [comment]
Launched an experiment on ARK. It'll be merged into the extension in a few weeks.

**Comment by alex4401** [comment]
Done in v1.2.0. Deployment pending on wiki.gg.
